net-dns/maradns/maradns-3.4.01.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-single-r1 systemd toolchain-funcs
+
+DESCRIPTION="A security-aware DNS server"
+HOMEPAGE="https://maradns.samiam.org/"
+SRC_URI="https://maradns.samiam.org/download/${PV%.*}/${PV}/${P}.tar.bz2"
+
+# The GPL-2 covers the init script, bug 426018.
+LICENSE="BSD-2 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~mips ~ppc ~x86"
+IUSE="authonly examples ipv6 systemd"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}
+	acct-group/maradns
+	acct-user/duende
+	acct-user/maradns
+	dev-lang/perl"
```

Comment:
Is perl only used during runtime?
It seems, that's is also used for building?
So it would need to go also into BDEPEND with EAPI=7.

Reply:
yes, one tool (csv1tocsv2.pl and it's sub) written on perl. It is not BDEPEND, it is clear RDEPEND